src/v0/util/constant.js

```diff
@@ -10,9 +10,79 @@ const API_CALL = 'api_call_count';
 
 const JSON_MIME_TYPE = 'application/json';
 
+const HttpStatus = {
```

Comment:
```suggestion
const HTTP_STATUS_MAP = {
```

In the future, this should become an enum.

Reply:
Some more alternatives I can think of:
- HTTP_STATUS_CODES
- HTTP_STATUSES